package.json

```diff
@@ -9,7 +9,7 @@
     "eslint": "^5.16.0",
     "eslint-config-recommended": "^4.0.0",
     "eslint-plugin-mocha": "^5.3.0",
-    "glob": "^7.1.3",
+    "glob": "^7.1.4",
```

Comment:
Side note, might not be really important, but you're upgrading `glob` too.

Reply:
yep, I should have mentioned this in the PR. We like to try and keep all the dev deps up-to-date. Updating this was a minor version and the tests passed and build was unchanged so it should be all good 👍 . Thanks for reviewing!

Reply:
> We like to try and keep all the dev deps up-to-date.

👍 

You can also use Dependabot for that, very handy (and just acquired by GitHub)! https://github.com/marketplace/dependabot-preview

Reply:
YES! I actually have that installed on all my own GitHub repos - and found out about your update to Apache Server Configs from the fork I have of html5-boilerplate :-)  Yes, we should install Dependabot on all the H5BP repos I think. But they need to be tested before merging - and should ideally be hooked up to CI for testing.

Reply:
We used to run greenkeeper and then that went crazy because the project was neglected for a few months in between active maintainers. I'm happy to look at something to automate the process.